core/src/main/java/org/frankframework/parameters/DateParameter.java

```diff
@@ -75,6 +76,14 @@ public void configure() throws ConfigurationException {
 			}
 		}
 
+		if(formatType != DateFormatType.XMLDATETIME) {
+			try {
+				dateFormat = new SimpleDateFormat(getFormatString());
```

Comment:
SimpleDateFormat is niet thread-safe dus dit zou ik niet doen. De DataParameter instance wordt gedeeld door threads.

Reply:
Ik had geprobeerd er een instant van te maken, dit is inderdaad niet handig zo